src/main/java/frc/robot/subsystems/Transfer.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap.mapTransfer;
+
+public class Transfer extends SubsystemBase {
+  TalonFX feederMotor;
+
+  /** Creates a new Transfer. */
+  public Transfer() {
+    feederMotor = new TalonFX(mapTransfer.TRANSFER_MOTOR_CAN, "rio");
```

Comment:
Remember to reset to factory defaults by creating and applying a TalonFXConfiguration